include/x86_64/drivers/serial.hpp

```diff
@@ -4,13 +4,15 @@
 #include <cstdlib/cstdint.h>
 
 namespace firefly::kernel::io {
+void serial(const char* msg);

```

Comment:
eh i dont agree with this, ngl. but when i finish buddy alloc ill refactor C-like stuff

Reply:
alright, I'll remove it.
I admit I haven't put a lot of thought into that one

Reply:
no it's ok for now, there's a lot of C-like things but it's ok

Reply:
C ftw >:D :1st_place_medal: